sor/src/main/java/com/bazaarvoice/emodb/sor/db/astyanax/CqlBlockedDataReaderDAO.java

```diff
@@ -722,9 +721,14 @@ protected Object getKeyForRow(Row row) {
 
         @Override
         protected ResultSet queryRowGroupRowsAfter(Row row) {
-            Range<RangeTimeUUID> columnRange = Range.greaterThan(new RangeTimeUUID(getChangeId(row)));
-            return columnScan(_placement, _placement.getBlockedDeltaTableDDL(), getKey(row),
-                    columnRange, true, _consistency);
+            Statement statement = selectDeltaFrom(_placement.getBlockedDeltaTableDDL())
```

Comment:
This is the main functionality change. The rest of this PR is just adding necessary information to the code path for enhanced logging.